build.properties.dist

```diff
@@ -33,6 +33,7 @@ drupal.admin.email = admin@example.com
 # Comma-separated list of demo users to create. The username and password will
 # be taken from the role.
 drupal.demo.users = administrator, moderator
+drupal.demo.authenticated_users = user
```

Comment:
This is not needed any more. I was thinking, maybe we can add "authenticated" to `drupal.demo.users`? So we can log in as a normal authenticated user? I'm not sure if that will work though.